lib/modules/manager/conan/readme.md

```diff
@@ -3,7 +3,7 @@
     The Conan package manager is disabled by default due to slowness in the Conan API.
     We recommend you only enable it for low volume experimental purposes until [issue #14170](https://github.com/renovatebot/renovate/issues/14170) is resolved.
 
-Renovate can upgrade dependencies in `conanfile.txt` or `conanfile.py` files.
+Renovate can upgrade dependencies in `conanfile.txt` or `conanfile.py` files and update `conan.lock` appropriately if present.
```

Comment:
This line may mislead people. What users normally expect is: if you update the package file, also update the lock file at the same time. Additionally, there is the concept of "lockFileMaintenance" which means "do a full refresh of the lock file" independently on any changes to the package file.

In this PR it appears to exit unless it's in lock file maintenance mode. Was that intentional?

Reply:
Yes, it was intentional, but I misunderstood the concept of "lockFileMaintenance", my bad. Thanks for the clarification. I pushed the fix, so please take a look at the code again :)